docs/en/04_Changelogs/4.0.0.md

```diff
@@ -1222,36 +1196,37 @@ See our ["Caching" docs](/developer-guides/performance/caching) for more details
  
 Before (`mysite/_config.php`):
 
-    :::php
-    Cache::add_backend(
-        'primary_memcached', 
-        'Memcached',
-        array(
-            'servers' => array(
-                'host' => 'localhost', 
-                'port' => 11211, 
-            )
+```php
+Cache::add_backend(
+    'primary_memcached', 
+    'Memcached',
+    array(
+        'servers' => array(
```

Comment:
You seem to be enforcing short array syntax (every other one of your updates used it), so might be good to have consistency here.